laplace-essay-on-probabilities.md

```diff
@@ -214,57 +214,57 @@ the sum of the possibilities of each favorable case. Let us clarify
 this principle by an example.
 
 Suppose we project into the air a broad and very thin coin whose two
-large opposed faces, which we name *cross* and *pillar*, be perfectly
-similar. We seek the probability of achieving *cross* at least once in
+large opposed faces, which we name *heads* and *tails*, be perfectly
```

Comment:
As a matter of taste I like a "light" translation doesn't try to transpose cultural elements, but admittedly I have no idea if cross and pillar were the actual referents for those faces of a coin at that time/place. I just looked in a dictionary...

Reply:
Yeah, croix et pile are the two faces of a coin (pile et face, nowadays), and "croix ou pile" is the old term for "pile our face". https://fr.wikipedia.org/wiki/Pile_ou_face